flaskbb.cfg

```diff
@@ -0,0 +1,285 @@
+# This file has been automatically generated on Thursday, 20. May 2021 at 00:45.
```

Comment:
Same for this file. See my comments above.

Reply:
Okay thanks